httptools/parser/cparser.pxd

```diff
@@ -1,139 +1,156 @@
-from libc.stdint cimport uint16_t, uint32_t, uint64_t
+from libc.stdint cimport int32_t, uint8_t, uint16_t, uint64_t
 
 
-cdef extern from "../../vendor/http-parser/http_parser.h":
-    ctypedef int (*http_data_cb) (http_parser*,
+cdef extern from "../../vendor/llhttp/include/llhttp.h":
```

Comment:
```suggestion
cdef extern from "llhttp.h":
```

This was a leftover issue - we should let the compiler find the right header files instead of hard-coding here - as we do support `--use-system-http-parser`.